modules/tinymce/src/plugins/paste/main/ts/core/Quirks.ts

```diff
@@ -83,7 +82,7 @@ const removeWebKitStyles = (editor: Editor, content: string, internal: boolean):
         }
 
         if (currentValue !== compareInput) {
-          outputStyles[webKitStyles[i]] = inputValue;
+          outputStyles[webKitStyles[i]] = compareInput;
```

Comment:
This was a deliberate change to avoid changing styles to hex; the aim is to just check whether the colours match, not force them to be hex if they do.

It's the same thing we did in the DOM code where we leave rgb alone now instead of converting it to hex.